cryptoxlib/clients/binance/BinanceClient.py

```diff
@@ -35,8 +35,18 @@ def _get_websocket_mgr(self, subscriptions: List[Subscription], startup_delay_ms
     async def ping(self) -> dict:
         return await self._create_get("ping", api_variable_path = BinanceClient.API_V3)
 
-    async def get_exchange_info(self) -> dict:
-        return await self._create_get("exchangeInfo", api_variable_path = BinanceClient.API_V3)
+    async def get_exchange_info(self, symbol: str = None, symbols: List[str] = None) -> dict:
```

Comment:
API preferably uses `Pair` class which is later transformed to symbol via `map_pair(...)`, see other methods.

I would also avoid having two parameters - `symbol` and `symbols` (or `pair` and `pairs`). Instead I'd have only `pairs` and if there is only one element, then use "?symbol=" in the path, if there are more, then use "?symbols=" exactly as you did below.

Apart from these two minor points, I like it, thanks!

Reply:
I am okay with that Pair change, but maybe just for sake of speed / efficiency the functionality should be split into separate named functions? I was up late (lol), the multiple if conditions aren't great IMO. I propose

- get_exchange_info (left as it is)
- get_symbol_info
- get_symbols_info

And avoid the if-based processing altogether. Thoughts?

Reply:
I made a separate PR so you can see the changes I proposed above, combined with general usage of the Pair type.

Reply:
Hi @nardew 

I made your original requested changes here, they've been tested. It is still one function with parameter usage as per your request. I may still use the alternate forms in some of my own code, but I update this PR for completion. 

- Uses Pair
- Handles 'symbols=' and 'symbol=' tags based on length of input list parameter, 'pairs'

Reply:
Hi @nkrsic , sorry for the late feedback but could not get to this earlier. Many thanks for the updates, I will add a unit test for the impacted method and publish it soon.